eggplant/membership/models/account.py

```diff
@@ -28,7 +30,7 @@ def __str__(self):
 
 
 class Account(models.Model):
-    number = models.PositiveSmallIntegerField()
+    number = models.AutoField(primary_key=True)
```

Comment:
yes, you're right, let's keep id, and for now keep number as null(?) or remove it completely and add it when needed(?)